lib/src/constants/constants.dart

```diff
@@ -1,4 +1,11 @@
-class HomeScreen {}
+import 'package:flutter/material.dart';
+
+class HomeScreen {
+  static const colorOfTextAndIndicator = Color(0xff8FC746);
```

Comment:
constants file like this should only include simple variable types

Reply:
i fixed it